tracer/src/Datadog.Tracer.Native/cor_profiler.cpp

```diff
@@ -2121,6 +2126,26 @@ int CorProfiler::GetProbesStatuses(WCHAR** probeIds, int probeIdsLength, debugge
     return 0;
 }
 
+//
+// Fault-Tolerant Instrumentation methods
+//
+void CorProfiler::ReportSuccessfulInstrumentation(ModuleID moduleId, int methodToken, const WCHAR* instrumentationVersion,
+    int products)
+{
+    const auto& instrumentationVersionString = shared::WSTRING(instrumentationVersion);
+    const auto instrumentingProducts = static_cast<InstrumentingProducts>(products);
+    const auto methodId = static_cast<mdMethodDef>(methodToken);
+    fault_tolerant::FaultTolerantTracker::Instance()->AddSuccessfulInstrumentationVersion(moduleId, methodId, instrumentationVersionString, instrumentingProducts, rejit_handler);
+}
+
+bool CorProfiler::ShouldHeal(ModuleID moduleId, int methodToken, const WCHAR* instrumentationVersion, int products)
+{
+    const auto& instrumentationVersionString = shared::WSTRING(instrumentationVersion);
```

Comment:
same

Reply:
Done in 9a92692d26bede161b4c38dab24df7c7395bc3bb.